models/stg_linkedin_ads__monthly_ad_analytics_by_country.sql

```diff
@@ -0,0 +1,55 @@
+{{ config(enabled=fivetran_utils.enabled_vars(['ad_reporting__linkedin_ads_enabled','linkedin_ads__using_monthly_ad_analytics_by_member_country'])) }}
+
+
+with base as (
+
+    select * 
+    from {{ ref('stg_linkedin_ads__monthly_ad_analytics_by_country_tmp') }}
+),
+
+macro as (
+
+    select
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(ref('stg_linkedin_ads__monthly_ad_analytics_by_country_tmp')),
+                staging_columns=get_monthly_ad_analytics_by_member_country_columns()
+            )
+        }}
+    
+        {{ fivetran_utils.source_relation(
+            union_schema_variable='linkedin_ads_union_schemas', 
+            union_database_variable='linkedin_ads_union_databases') 
+        }}
+
+    from base
+),
+
+fields as (
+    
+    select 
+        source_relation,
+        {{ linkedin_source.date_from_month_string('date_month') }} as date_month, --Renamed in macros/get_monthly_ad_analytics_by_member_country_columns to avoid reserved word error.
+        campaign_id,
+        member_country,
```

Comment:
```suggestion
        member_country as member_country_geo_id,
```
So it's clear it's a FK?

Reply:
Good idea